src/main/resources/public/js/log.provider.js

```diff
@@ -24,5 +24,13 @@
       };
       return $delegate;
     }
+
+    function getParameterByName(name) {
```

Comment:
Inspired by http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/3855394#3855394